docs/overview.md

```diff
@@ -3,35 +3,48 @@ Spank plugins for Slurm to support quantum resources
 
 ## Content
 
+- [Context](#content)
 - [Definitions](#definitions)
   - [QPU](#qpu)
   - [Quantum computer](#quantum-computer)
+  - [IBM Quantum Platform](#ibm-quantum-platform)
```

Comment:
should we move it to some vendor specific folder or section, because it is only for IBM?

Reply:
I like the idea, let me re-shuffle